nc-games/src/components/ReviewById/ReviewById.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { useEffect } from "react"
+import { useState } from "react"
+import { useParams } from "react-router-dom"
+import { getReviewById } from "../../utils/api"
+
+
+
+export const ReviewById = () =>{
+    const { review_id } = useParams()
+    const [reviewNumber, setReviewNumber]= useState(1)
+
+
+    useEffect(()=>{
+        getReviewById(review_id)
+        .then((thisReview)=>{
+            setReviewNumber(thisReview)
+        })
+    },[review_id]) 
+    
```

Comment:
i've quite confused about `reviewNumber` here. It sounds like a number - and starts as `1` - but you set it to the review itself later on!

Would be more readable if you kept the data types more consistent (and maybe rename the state if it helps)